test_themes/__init__.py

```diff
@@ -10,11 +10,8 @@ def post_init_hook(cr, registry):
     ''' Create a new website for each theme and install the theme on it. '''
     env = api.Environment(cr, SUPERUSER_ID, {})
     IrModule = env['ir.module.module']
-
-    themes = IrModule.search([
-        ('category_id', 'child_of', env.ref('base.module_category_theme').id),
-    ], order='name')
-    exclude_list = ['_common', '_blog', '_sale']
+    themes = IrModule.search(IrModule.get_themes_domain(), order='name')
+    exclude_list = ['_common']
```

Comment:
Mmh, did not you say in the community commit that excluding theme_common is already done by `get_themes_domain`?